### PR TITLE
Review grammar in Developer guide and Howto pages

### PR DESCRIPTION
**Pull request by akohlmey**
**Summary**

This applies (some) grammar fixes suggested by languagetool.org for the first part of the Developer guide section of the manual and some of the Howto pages.

**Author(s)**

Axel Kohlmeyer, Temple U

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Post Submission Checklist**

- [x] The feature or features in this pull request is complete
- [x] Licensing information is complete
- [x] Corresponding author information is complete
- [x] The source code follows the LAMMPS formatting guidelines
- [x] Suitable new documentation files and/or updates to the existing docs are included
- [x] The added/updated documentation is integrated and tested with the documentation build system

### CODE REVIEW

**Review by sjplimp — APPROVED** [review]

File: doc/src/Developer_org.rst [hunk 1]
```diff
@@ -102,42 +103,44 @@ LAMMPS while passing it the command line flags and input script. It
...
-*hybrid* class that takes the place of the one allowed instance and then
-manages and forwards calls to the corresponding sub-styles for the
-designated subset of atoms or data.  The composite class may also have
-lists of class instances, e.g. Modify handles lists of compute and fix
-styles, while Output handles a list of dump class instances.
+*hybrid* class instance that acts as a proxy, and manages and forwards
+calls to the corresponding sub-style class instances for the designated
+subset of atoms or data.  The composite class may also have lists of
+class instances, e.g. ``Modify`` handles lists of compute and fix
+styles, while ``Output`` handles a list of dump class instances.
```
Comment (athomps) [inline doc/src/Developer_org.rst:143]: Should we make all LAMMPS class names ``inline literal``?

File: doc/src/Howto_bpm.rst [hunk 2]
```diff
@@ -3,16 +3,16 @@ Bonded particle models
 
 The BPM package implements bonded particle models which can be used to
 simulate mesoscale solids.  Solids are constructed as a collection of
-particles which each represent a coarse-grained region of space much
-larger than the atomistic scale. Particles within a solid region are
+particles, which each represent a coarse-grained region of space much
+larger than the atomistic scale.  Particles within a solid region are
```
Comment (athomps) [inline doc/src/Howto_bpm.rst:7]: Oh-oh, are we going to have to put two spaces after *every* period?

Comment (stanmoore1) [reply doc/src/Howto_bpm.rst:7]: Please no...

**Review by athomps — APPROVED** [review]
I approve. See optional suggestion in comment on ``inline literal``

File: doc/src/Developer_org.rst (hunk 1 above)
Comment (akohlmey) [reply doc/src/Developer_org.rst:143]: In an ideal world, those would be documented with Doxygen comments and their API would be imported into the Developer's Guide reference section and then those would be referred to with ":cpp:class:`Modify <LAMMPS_NS::Modify>`" . But we are still a long way away from that. When I see it, I use "``" markup for files, class names/functions, and defines or CMake variables. To make them stand out more. But I don't see this as a "must have" change.

File: doc/src/Howto_bpm.rst (hunk 2 above)
Comment (akohlmey) [reply doc/src/Howto_bpm.rst:7]: No. Those are usually inserted automatically by emacs when I reindent or rewrap a block of text. Same as for the class name, this is a "can have" but not a "must have" change.

File: doc/src/Developer_org.rst (hunk 1 above)
Comment (akohlmey) [reply doc/src/Developer_org.rst:143]: We have arrived at over 2800 pages (in PDF format) in the manual, so there is a lot of "legacy text" around that was added before we started to pay closer attention to consistency.  Keep in mind that most files were auto-converted to .rst markup from txt2html markup and that had its issues with some corner cases (even of we have by now eliminated 99.99% of those, at that ratio and with 2800 pages, there would still be 28 pages with a leftover issue)

**Pull request merged**